src/bindings/python/requirements_nvidia.txt

```diff
@@ -0,0 +1,3 @@
+cuda-python==11.8.2 ; platform_system == "Linux" and platform_machine == "x86_64"
```

Comment:
Ideally we would not require these specific versions. Do we specify this because this is validated, or are these hard requirements? With this it is impossible to install openvino[nvidia] and PyTorch 2.0.* at the same time:

```
ERROR: Cannot install openvino[nvidia]==2023.1.0 and torch==2.0.1 because these package versions have conflicting dependencies.

The conflict is caused by:
    torch 2.0.1 depends on nvidia-cudnn-cu11==8.5.0.96; platform_system == "Linux" and platform_machine == "x86_64"
    openvino[nvidia] 2023.1.0 depends on nvidia-cudnn-cu11==8.6.0.163; (platform_system == "Linux" and platform_machine == "x86_64") and extra == "nvidia"
```

When I did not specify a PyTorch version these pinnings for CUDA packages gave me PyTorch 1.12.

Reply:
We require such specific versions because we built NVIDIA plugin based on them 
https://github.com/openvinotoolkit/openvino_contrib/tree/master/modules/nvidia_plugin#prerequisites
And yeah also it because of validation.

Good point about installing PyTorch 2.0* in parallel, since our openvino_dev wheel have following deps for extra [pytorch]
```
torch (>=1.8.1)
```